bcgov/mds
# Change value shown for Inspector Responsible in Incidents Table

# Main

- Fix the value shown in Incidents Table for Inspector responsible

# Other
- This change is modifying a previous PR https://github.com/bcgov/mds/pull/2008
- Original ticket: https://bcmines.atlassian.net/browse/MDS-4149

# How to test
-  In Core, select a mine. Then click on Oersight > Incidents & Investigation
-  Create several Mines Incidents:
![image](https://user-images.githubusercontent.com/10526131/164329563-e69a6f8f-3e1c-49d4-8d9f-5d08e3d26c54.png)
-  Filter by Inspector Responsible:
![image](https://user-images.githubusercontent.com/10526131/164329800-8353defc-9193-4d8a-bbec-af6d7dd7e868.png)
-  Check mine_incident.responsible_inspector_party_guid corresponds to the record shown in the UI.

# Notes
https://bcmines.atlassian.net/browse/MDS-4263
